### PR TITLE
task-5: added S3 upload handler for csv

### PR DESCRIPTION
**Pull request by alexanderteplov**
all, except unit tests

### CODE REVIEW

File: import-service/src/functions/importFileParser/handler.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  S3Client,
+  GetObjectCommand,
+  CopyObjectCommand,
+  DeleteObjectCommand,
+} from '@aws-sdk/client-s3';
+import csv from 'csv-parser';
+
+import { formatJSONResponse } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import stream from 'stream';
```
Comment (maxm-dev) [inline import-service/src/functions/importFileParser/handler.ts:11]: You can import promisified version of all stream from `stream/promises`. Described [here](https://nodejs.org/api/stream.html#streams-promises-api).

**Review by maxm-dev — APPROVED** [review]
Well done!

File: import-service/src/functions/importFileParser/handler.ts (hunk 1 above)
Comment (alexanderteplov) [reply import-service/src/functions/importFileParser/handler.ts:11]: Unfortunately, it works only from version 15.0.0 of NodeJs

Comment (maxm-dev) [reply import-service/src/functions/importFileParser/handler.ts:11]: You are right; I've missed it. We are using v16 on the project 🙂 .

**Pull request merged**